ts/components/ui/utils/baseListItems.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import { Pressable } from "react-native";
+import Animated from "react-native-reanimated";
+import { WithTestID } from "../../../types/WithTestID";
+import { IOListItemStyles } from "../../core/variables/IOStyles";
+import { useDefaultListItemSpringAnimation } from "./hooks";
+
+export type PressableBaseProps = WithTestID<{
+  accessibilityLabel?: string;
+  onPress?: () => void;
+}>;
+export const PressableListItemBase = ({
```

Comment:
The `useDefaultListItemSpringAnimation()` hook is already a good abstraction. Do we really need another abstraction for the `Pressable` component in another file?

Reply:
seeing that this structure is reused in multiple listItems in the DS, I believe it would be good to just reuse it at least in future ones to make the files more readable.
Do you really think it'd add much more abstraction? I could add a `docString` to make it clearer to possible readers